tests/page/wheel.spec.ts

```diff
@@ -20,15 +20,23 @@ it.skip(({ isElectron, browserMajorVersion, isAndroid }) => {
   // Old Electron has flaky wheel events.
   return (isElectron && browserMajorVersion <= 11) || isAndroid;
 });
+
+let deltaScale = 1;
+
+it.beforeAll(async ({ browserMajorVersion, browserName, platform }) => {
+  if (browserName === 'chromium' && browserMajorVersion >= 102 && platform === 'darwin')
```

Comment:
Can you add a comment that explains why this divergence is needed (or link to relevant CR bits)?

Reply:
I filed an issue for you: https://bugs.chromium.org/p/chromium/issues/detail?id=1324819
As long as scrollTop is good, we are Ok.

Reply:
Thank you!